### PR TITLE
fibonacciExample in C#

### PR DESCRIPTION
**Pull request by JoonasM**
Got rejected in the fibonacci issue with this one and I haven't completed a single successfull pull request yet, so I hope this finally scores!

### CODE REVIEW

File: src/C#/pablopickaxo-fibonacci.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Simple solution for printing fibonacci sequence!
+
+using System;  
+
+  public class FibonacciExample
```
Comment (jazibjafri) [inline src/C#/pablopickaxo-fibonacci.cs]: Rename the class to match the name in filename, or vice versa

**Comment by jazibjafri** [comment]
@pablopickaxo rename the `C#` folder to `c-sharp`
And *remove* the `.exe` file, only add the code

**Review by jazibjafri — CHANGES_REQUESTED** [review]

File: src/C#/pablopickaxo-fibonacci.cs (hunk 1 above)
Comment (JoonasM) [reply src/C#/pablopickaxo-fibonacci.cs]: Will do!

**Comment by JoonasM** [comment]
Fixes done!

**Comment by jazibjafri** [comment]
@pablopickaxo Thanks for the PR! :rocket:

**Review by jazibjafri — APPROVED** [review]

**Pull request merged**